engine/class_modules/sc_death_knight.cpp

```diff
@@ -6694,6 +6735,14 @@ struct unholy_blight_dot_t : public death_knight_spell_t
     death_knight_spell_t::impact( state );
 
     td( state->target ) -> debuff.unholy_blight -> trigger();
+
+    if ( p() -> legendary.superstrain -> ok() )
```

Comment:
I attached the application of the off specs dots here as the tick callback doesn't have state as a parameter and I wanted to make sure the target was set explicitly since the offspec dots aren't child actions of UB like VP is.  Maybe they should be?

Reply:
Well it's more that unlike VP, they're not applied in aoe.

Reply:
I think adding them to blight's dot component's impact override is good, moving VP application there might also be a good idea.